app/src/main/java/org/simple/clinic/contactpatient/ContactPatientEffect.kt

```diff
@@ -47,3 +47,7 @@ data class CancelAppointment(
 data class MarkPatientAsMovedToPrivate(
     val patientUuid : UUID
 ): ContactPatientEffect()
+
+data class RemoveAppointment(
```

Comment:
We already have a `CancelAppointment` effect. Do we need a new effect here, when that one accomplishes the same task?